Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.11
+
+WORKDIR /app
+COPY . .
```

Comment:
This is fine at this stage in the product, but please note that __pycache__ will be generated once the container is running and that this requires your Python process to have write access to the file system, which is not really best practice. Ideally, you want to pre-populate your __pycache__ and add a `USER` directive to switch to an unprivileged user inside the container. Unless you want to deep-dive into container security best practices, the best way to handle this now is probably to create an issue referencing this comment and we can come back to it later.

Reply:
Issue created